tests/jsr_project/mod.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright 2018-2024 the Deno authors. MIT license.
 
-import { parse } from "jsr:@std/csv/parse";
+import { parse } from "jsr:@std/csv@1.0.6/parse";
```

Comment:
I added a version specifier to fix a linter error. The version used is based on the latest `@std/csv` version specified in deno.lock.